sdk/keyvault/keyvault-certificates/tests/mergeAndImport.test.ts

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { CertificatesClient } from "../src";
+import { retry } from "./utils/recorder";
+import { env } from "@azure/test-utils-recorder";
+import { authenticate } from "./utils/testAuthentication";
+import TestClient from "./utils/testClient";
+import { stringToUint8Array } from "./utils/crypto"
+import { SecretsClient } from "@azure/keyvault-secrets";
+import { ClientSecretCredential } from "@azure/identity";
+
+describe("Certificates client - merge and import certificates", () => {
+  const prefix = `merge${env.CERTIFICATE_NAME || "CertificateName"}`;
+  let suffix: string;
+  let client: CertificatesClient;
+  let testClient: TestClient;
+  let recorder: any;
+  let keyVaultUrl: string;
+  let credential: ClientSecretCredential;
+  let secretsClient: SecretsClient;
+
+  beforeEach(async function() {
+    const authentication = await authenticate(this);
+    suffix = authentication.suffix;
+    client = authentication.client;
+    testClient = authentication.testClient;
+    recorder = authentication.recorder;
+    keyVaultUrl = authentication.keyVaultUrl;
+    credential = authentication.credential;
+    secretsClient = new SecretsClient(keyVaultUrl, credential);
+  });
+
+  afterEach(async function() {
+    recorder.stop();
+  });
+
+  // The tests follow
+
+  it("can import a certificate from a backup", async function() {
+    const certificateName = testClient.formatName(`${prefix}-${this!.test!.title}-${suffix}`);
+    const certificateNames = [`${certificateName}0`, `${certificateName}1`];
+    await client.createCertificate(certificateNames[0], {
+      certificatePolicy: {
+        issuerParameters: { name: "Self" },
+        x509CertificateProperties: { subject: "cn=MyCert" }
+      }
+    });
+    const certificateSecret = await retry(async () => secretsClient.getSecret(certificateNames[0]));
+    const base64EncodedCertificate = certificateSecret.value!;
+    await client.importCertificate(certificateNames[1], base64EncodedCertificate);
+
+    for (const name of certificateNames) {
+      await testClient.flushCertificate(name);
+    } 
+  });
+
+  // TODO: Certificate can not be read. Please check if certificate is valid.
```

Comment:
is this TODO still valid? If so, we may want to hold off on landing a test for merge until we can confirm